drivers/can/can_loopback.c

```diff
@@ -66,12 +66,16 @@ static void tx_thread(void *arg1, void *arg2, void *arg3)
 	struct can_loopback_data *data = dev->data;
 	struct can_loopback_frame frame;
 	struct can_loopback_filter *filter;
+	int ret;
 
 	ARG_UNUSED(arg2);
 	ARG_UNUSED(arg3);
 
 	while (1) {
-		k_msgq_get(&data->tx_msgq, &frame, K_FOREVER);
+		ret = k_msgq_get(&data->tx_msgq, &frame, K_FOREVER);
+		if (ret < 0) {
+			continue;
```

Comment:
Nit: add a debug log here.

Reply:
Would this do: `LOG_DBG("Pend on TX queue returned without valid frame (err %d)", ret);`?

Reply:
Something along those lines would be fine, thanks.